avatar_triple_controller:
    type: avatar_dual_controllers/AvatarTripleController
    left:
        arm_id: panda_left
        joint_names:
            - panda_left_joint1
            - panda_left_joint2
            - panda_left_joint3
            - panda_left_joint4
            - panda_left_joint5
            - panda_left_joint6
            - panda_left_joint7
    right:
        arm_id: panda_right
        joint_names:
            - panda_right_joint1
            - panda_right_joint2
            - panda_right_joint3
            - panda_right_joint4
            - panda_right_joint5
            - panda_right_joint6
            - panda_right_joint7
    top:
        arm_id: panda_top
        joint_names:
            - panda_top_joint1
            - panda_top_joint2
            - panda_top_joint3
            - panda_top_joint4
            - panda_top_joint5
            - panda_top_joint6
            - panda_top_joint7

avatar_dual_controller:
    type: avatar_dual_controllers/AvatarDualController
    left:
        arm_id: panda_left
        joint_names:
            - panda_left_joint1
            - panda_left_joint2
            - panda_left_joint3
            - panda_left_joint4
            - panda_left_joint5
            - panda_left_joint6
            - panda_left_joint7
    right:
        arm_id: panda_right
        joint_names:
            - panda_right_joint1
            - panda_right_joint2
            - panda_right_joint3
            - panda_right_joint4
            - panda_right_joint5
            - panda_right_joint6
            - panda_right_joint7
            
avatar_single_controller:
    type: avatar_dual_controllers/AvatarSingleController
    right:
        arm_id: panda_right
        joint_names:
            - panda_right_joint1
            - panda_right_joint2
            - panda_right_joint3
            - panda_right_joint4
            - panda_right_joint5
            - panda_right_joint6
            - panda_right_joint7
